website/static/js/share/index.js

```diff
@@ -87,6 +92,14 @@ ShareApp.controller = function() {
 
     });
 
+    m.request({
+        method: 'GET',
+        url: '/api/v1/share/documents/' // TODO where will the postgres API live??
+    }).then(function(data) {
+        self.vm.rawNormedLoaded(true);
+    }, function(err) {
+        // blank, but needed to handle missing rawNormed API
```

Comment:
OK, thank you both for the clarification. I think the empty error handler is find. Maybe we just make the code comment more clear. The blank error handler isn't actually "needed" to handle the error returned from the API. It would be good to explicitly state that we expect an error response when ____ happens.